source/reference/modules/deployer/index.rst

```diff
@@ -355,6 +354,122 @@ the steps described in :ref:`debugging-deployer-issues`
+          remoteBinaryPathPatterns: &remoteBinaryPathPatterns
+            # HTTP/HTTPS URLs are only indexed if they contain the protocol (http:// or https://). Protocol relative
+            # URLs (like //mydoc.pdf) are not supported since the protocol is unknown to the back-end indexer.
+            - ^(http:|https:)//.+$
+            - ^/remote-assets/.+$
+
+""""""""""""""
+Jacket Pattern
+""""""""""""""
+The ``metadataPathPatterns`` in the configuration determines if a document should be indexed with the metadata of the
+object that points to it (a so-called "jacket"). The deployer will re-index the jacket and the document together
+whenever the jacket is updated. See :ref:`jacket` for more information.
+
+.. code-block:: yaml
+    :caption: *CRAFTER_HOME/bin/crafter-deployer/config/base-target.yaml*
+    :linenos:
+    :emphasize-lines: 8-9
+
+    target:
+    ...
+      search:
+        openSearch:
+        ...
+        binary:
+          ...
+          # The regex path patterns for the metadata ("jacket") files of binary/document files
+          metadataPathPatterns:
+            - ^/?site/documents/.+\.xml$
+
+""""""""""""""""
+Indexing Failure
+""""""""""""""""
+When a document cannot be indexed:
+
+- The Deployer skips the document and leaves an error in the logs.
+- The document is not re-processed unless it is updated/re-deployed or the re-process API is called.
+
+The default behavior when a document cannot be indexed is that the Deployer logs the error and moves on. Process commits
+files are updated and the Deployer never revisits the indexing unless a future publish requires it to, or, a
+re-process API is called, such as the `deployTarget <../../_static/api/studio.html#tag/target/operation/deployTarget>`__ API
+
+If a catastrophic exception interferes with the update of the processed commits file then the Deployer will re-process
+the commits from the last commit presently in the processed commits file.
+
+A few installations disable this behavior in the :ref:`GitDiffProcessor <deployer-git-diff-processor>` (via the
+``target.yaml``) that always moves forward.
+
+In these configurations, an additional processor is added to the end of the processor chain called the
+:ref:`Git Update Commit ID processor <deployer-git-update-commit-id-processor>` which only updates this ID on successful
+completion of the processor chain.
+
+Note that using this model can lead to A LOT of re-processing if there is no additional circuit breaker logging added
+for error conditions that do not resolve themselves.
+
+If the deployment as a whole cannot be completed due to a catastrophic exception, then all content including documents
+will be re-processed until the deployment succeeds. By default the Git Diff process is configured to update the processed
+commits regardless of success or failure. Some deployments set this to false and force the processor chain to be
```

Comment:
This repeats the paragraphs above.

These paragraphs regarding the different configuration should be pulled into a single paragraph that describes this approach and details the pros and cons.